pkg/dipper/encryption.go

```diff
@@ -26,10 +26,16 @@ func DecryptAll(rpc RPCCaller, from interface{}) {
 			return "ENC[" + parts[1] + "]", true
 		}
 
-		data := []byte(parts[1])
-		decoded, err := base64.StdEncoding.DecodeString(string(data))
-		if err != nil {
-			Logger.Panicf("encrypted data shoud be base64 encoded")
+		var decoded []byte
+
+		if parts[1][0] == '"' {
+			decoded = []byte(strings.Trim(parts[1], "\""))
+		} else {
+			var err error
+			decoded, err = base64.StdEncoding.DecodeString(parts[1])
+			if err != nil {
+				Logger.Panicf("encrypted data shoud be base64 encoded")
+			}
```

Comment:
this whole routine seems exactly the same as the one in `internal/service`? should it be a common function?

Reply:
Merged them.